module/VuFind/src/VuFind/Search/Factory/BlenderBackendFactory.php

```diff
@@ -0,0 +1,185 @@
+<?php
+
+/**
+ * Factory for Blender backend.
+ *
+ * PHP version 7
+ *
+ * Copyright (C) Villanova University 2013.
+ * Copyright (C) The National Library of Finland 2013-2019.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Search
+ * @author   David Maus <maus@hab.de>
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     http://vufind.org   Main Site
+ */
+namespace VuFind\Search\Factory;
+
+use Interop\Container\ContainerInterface;
+use Laminas\EventManager\EventManager;
+use Laminas\ServiceManager\Factory\FactoryInterface;
+use VuFind\Search\Solr\DeduplicationListener;
+use VuFindSearch\Backend\Blender\Backend;
+use VuFindSearch\Backend\Solr\Backend as SolrBackend;
+
+/**
+ * Factory for Blender backend.
+ *
+ * @category VuFind
+ * @package  Search
+ * @author   David Maus <maus@hab.de>
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     http://vufind.org   Main Site
+ */
+class BlenderBackendFactory implements FactoryInterface
+{
+    /**
+     * Superior service manager.
```

Comment:
With ServiceManager v3+, there is no longer a concept of a superior service manager. We should probably string-replace this comment across a whole bunch of files to just be "Service manager". Might even be better to rename serviceLocator to container... but maybe I'm creating a lot of pointless work by that suggestion.

Reply:
Renamed.